scripts/rfid_trigger_play.py

```diff
@@ -0,0 +1,714 @@
+conf = conf_read_bash([path_config_global, path_config_debug, path_config_rfid])
+
+###################################
+# parse variables from command line
+parser = argparse.ArgumentParser()
+parser.add_argument('-i', '--cardid')
+parser.add_argument('-c', '--command')
+parser.add_argument('-d', '--dir')
+parser.add_argument('-v', '--value')
+args = parser.parse_args()
+args_main = vars(args)
+logger.debug('arguments passed on to script:')
+logger.debug(args_main)
+
+# Set the date and time of now
+now_string = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
+# create the configuration file from sample - if it does not exist
+if Path(path_config_rfid).is_file():
+    # file exists
+    logger.debug("rfid_trigger_play.conf exists")
+else:
+    # file does NOT exist
+    logger.debug("rfid_trigger_play.conf does NOT exist, copying from settings dir")
+    copyfile(path_dir_settings + "/rfid_trigger_play.conf.sample", path_config_rfid)
+
+# Read config files for global, debug and rfid
+conf = {}
+# global config
+with open(path_config_global) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
+# debugging config
+with open(path_config_debug) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
+# debugging rfid
+with open(path_config_rfid) as myfile:
+    for line in myfile:
+        if not line.lstrip().startswith('#'):
+            name, var = line.partition("=")[::2]
+            conf[name.strip()] = var.strip()
+# strip " off values in dictionary conf
+conf = {k: v.strip('"') for (k, v) in conf.items()}
+if conf['DEBUG_rfid_trigger_play_sh'] == "TRUE":
+    logger.debug('configuration found in conf files for global.conf, debugLogging.conf, rfid_trigger_play.conf:')
+    logger.debug(conf)
+
+#####################################################################
+# We might have a card ID, we might not (only folder -d)
+
+if(args_main['cardid']):    
+    if conf['DEBUG_rfid_trigger_play_sh'] == "TRUE":
+        logger.debug("Card ID given: '%s'" % (args_main['cardid']))
+
+    # Write info to file, making it easer to monitor cards
+    with open(path_txt_latestID, "w") as f:
+        f.write("Card ID '" + args_main['cardid'] + "' was used at '" + now_string + "'.")
+    with open(path_dir_settings + "/Latest_RFID", "w") as f:
+        f.write(args_main['cardid'])
+    
+    # If the input is of 'special' use, don't treat it like a trigger to play audio.
+    # Special uses are for example volume changes, skipping, muting sound.
+    # NOTE: we need to check if the key exists, because older installations might not have
+    # the key in their copied config file. If we don't, the script will throw an error.
+    if('CMDSHUFFLE' in conf):
+        if(args_main['cardid'] == conf['CMDSHUFFLE']):
```

Comment:
Here and below.. how about writing it into a combined condition? And it seams that Python does not necessarily require `( )` around the condition statement

```
if 'CMDSHUFFLE' in conf and args_main['cardid'] == conf['CMDSHUFFLE']:
```

Reply:
I might be mistaken, but Python should not do this.. if you have

`if A and B` and `A = True`, it shouldn't matter what `B` is. But I don't have much experience with Python, so someone else might speak about it.